packages/richtext-lexical/src/utilities/buildInitialState.ts

```diff
@@ -75,6 +76,7 @@ export async function buildInitialState({
         data: blockNode.fields,
         fields: (context.fieldSchemaMap.get(schemaFieldsPath) as any)?.fields,
         fieldSchemaMap: context.fieldSchemaMap,
+        fullDataOverride: context.documentData,
```

Comment:
I don't love the name of this property. It feels like `documentData` above is accurate, but `fullDataOverride` kind of throws me for a loop.